framework/src/org/apache/cordova/PluginEntry.java

```diff
@@ -52,6 +52,10 @@ public PluginEntry(String service, CordovaPlugin plugin) {
         this(service, plugin.getClass().getName(), true, plugin);
     }
 
+    public PluginEntry(String service, CordovaPlugin plugin, boolean onload) {
```

Comment:
Could you add a JavaDoc comment for this overload? Thanks!

Reply:
@jblejder would you be able to add in the JavaDoc comment?

You can use the below `public PluginEntry` overloaded method's comment block as an example for your overloaded method.

After this PR is updated, I can merge it in for next release.